src/lib/header-bidding/prebid/appnexus.ts

```diff
@@ -36,23 +35,18 @@ const getAppNexusInvCode = (sizes: HeaderBiddingSize[]): string | undefined => {
 	}
 };
 
-const getAppNexusDirectPlacementId = (
-	sizes: HeaderBiddingSize[],
-	isInFrontsBannerVariant: boolean,
-): string => {
+const getAppNexusDirectPlacementId = (sizes: HeaderBiddingSize[]): string => {
 	if (isInAuOrNz()) {
 		return '11016434';
 	}
 
 	const defaultPlacementId = '9251752';
 	switch (getBreakpointKey()) {
 		case 'D':
-			if (isInFrontsBannerVariant || includeBillboardsInMerchHigh()) {
```

Comment:
Note that `includeBillboardsInMerchHigh()` and `containsBillboardNotLeaderboard()` will always resolve to the same result. They are both true if and only if billboard is in `sizes`.